src/rgw/rgw_zone.h

```diff
@@ -274,11 +274,12 @@ struct RGWZonePlacementInfo {
   rgw_pool data_extra_pool; /* if not set we should use data_pool */
   RGWZoneStorageClasses storage_classes;
   rgw::BucketIndexType index_type;
+  bool inline_data;
 
-  RGWZonePlacementInfo() : index_type(rgw::BucketIndexType::Normal) {}
+  RGWZonePlacementInfo() : index_type(rgw::BucketIndexType::Normal), inline_data(true) {}
 
   void encode(bufferlist& bl) const {
-    ENCODE_START(7, 1, bl);
+    ENCODE_START(8, 1, bl);
```

Comment:
due to these encoding changes, this needs to go to quincy first so there's an upgrade path

Reply:
makes sense, thanks for reviewing @cbodley

Reply:
I'll create the quincy backport after the corresponding PR is merged in main